2022-02-26 13:50:38,340 - base_log.py[line:31] - ERROR: 执行用例11的时候报错:True is not false
